docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -0,0 +1,149 @@
+[[get-snapshot-api]]
+=== Get snapshot API
+++++
+<titleabbrev>Get snapshot</titleabbrev>
+++++
+
+Returns information about one or more snapshots from one or more registered repositories.
+
+////
+[source,console]
+----
+PUT /_snapshot/my_repository
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+
+PUT /_snapshot/my_repository/my_snapshot?wait_for_completion=true
+
+PUT /_snapshot/my_repository/snapshot_2?wait_for_completion=true
+----
+// TESTSETUP
+////
+
+[source,console]
+----
+GET /_snapshot/my_repository/my_snapshot
+----
+
+[[get-snapshot-api-request]]
+==== {api-request-title}
+
+`GET /_snapshot/<repository>/<snapshot>`
+
+[[get-snapshot-api-desc]]
+==== {api-description-title}
+
+Use the get snapshot API to return information about one or more snapshots, including start and end time, version of {es} that created the snapshot, the list of included indices, the current state of the snapshot, and the list of failures that occurred during the snapshot.
+
+[[get-snapshot-api-path-params]]
+==== {api-path-parms-title}
+
+`<repository>`::
+(Required, string)
+Comma-separated list of snapshot repository names used to limit the request.
+Wildcard (`*`) expressions are supported.
++
+To get information about all snapshot repositories registered in the
+cluster, omit this parameter or use `*` or `_all`.
+
+`<snapshot>`::
+(Required, string)
+Comma-separated list of snapshot names to return. Also accepts wildcards (`*`).
++
+* To get information about all snapshots in a registered repository, use a wildcard (`*`) or `_all`.
+* To get information about any snapshots that are currently running, omit this parameter and use `_current`.
```

Comment:
> * To get information about any snapshots that are currently running, omit this parameter and use `_current`.

Is this saying that you can omit this parameter _or_ use `_current`... or just use `_current`?

Reply:
Just use `_current`. I wanted to indicate that you should use `_current` in place of the snapshot name, but perhaps that inclusion is more confusing than helpful.